about.html

```diff
@@ -0,0 +1,69 @@
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="about-styles.css">
+</head>
+<body>
+  <div class="navbar">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@700&display=swap" rel="stylesheet">
+    <img src="mustacchio.jpg" alt="Mustacchio Logo">
+    <br>
+    <br>
+   <div class="buttons"><a href="./index.html">HOME</a>
+    <a href="./about.html">ABOUT</a>
+    <a href="./gallery.html">GALLERY</a>
+    <a href="./blog.html">BLOG</a>
+    <a href="./contact.html">CONTACT</a>
+  </div>
+  </div>
+
+  <div class="grid-container">
+    <section class="start"><h1>MUSTACCHIO STARTED</h1>
+      <link rel="preconnect" href="https://fonts.gstatic.com">
+      <link href="https://fonts.googleapis.com/css2?family=Changa:wght@700&display=swap" rel="stylesheet">
```

Comment:
yeah you can put all font links in the head tag and then use font-family: in css